pkg/controller/volume/expand/expand_controller.go

```diff
@@ -171,12 +171,6 @@ func (expc *expandController) deletePVC(obj interface{}) {
 }
```

Comment:
The check above misses deletion tombstone final state unknown objects and means they live in the resize map forever. @gnufied

Reply:
I do not understand - the resizeMap gets reset to empty on each iteration of loop - https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/expand/cache/volume_resize_map.go#L128

Reply:
deletePVC may be passed a cache.DeletedFinalStateUnknown object, not a v1.PersistentVolumeClaim... it needs to handle that

Reply:
yes, the controller should handle DeletedFinalStateUnknown, however in a separate PR (or issue).
@gnufied, would you mind fixing this?